src/core_plugins/i18n/server/i18n/index.js

```diff
@@ -0,0 +1,35 @@
+import  i18n from './i18n';
```

Comment:
Actually, now that I think about it, this entire file could be boiled down to one line:

```
export { getRegisteredLocaleTranslations, getRegisteredTranslationLocales, registerTranslations } from './i18n';
```

The documentation can go in the i18n.js file